code/integration-tests/local-integration-tests/src/helpers.rs

```diff
@@ -50,23 +51,19 @@ pub fn sibling_account() -> AccountId {
 }
 
 /// assert amount is supported deposit amount and is above it
-pub fn assert_above_deposit<AssetsRegistry: AssetRatioInspect<AssetId = CurrencyId>>(
+pub fn assert_above_deposit<
+	AssetsRegistry: AssetRatioInspect<AssetId = CurrencyId>
+		+ AssetExistentialDepositInspect<AssetId = CurrencyId, Balance = Balance>,
+>(
 	asset_id: CurrencyId,
 	amount: Balance,
 ) -> Balance {
-	assert!(
-		PriceConverter::<AssetsRegistry>::get_price_inverse(
-			asset_id,
-			NativeExistentialDeposit::get()
-		)
-		.unwrap() <= amount
-	);
+	assert!(multi_existential_deposits::<AssetsRegistry>(&asset_id) <= amount);
 	amount
 }
 
 /// weigh enough to handle any XCMP message
 pub fn enough_weight() -> u128 {
-	let this_liveness_native_amount = BaseXcmWeight::get() as u128 +
-		100 * UnitWeightCost::get() as Balance * MaxInstructions::get() as Balance;
-	this_liveness_native_amount
+	BaseXcmWeight::get() as u128 +
```

Comment:
Can we explain where this formula comes from?

Reply:
from xcm configuration